pipeline/install-prokka.sh

```diff
@@ -6,41 +6,41 @@ cd /mnt
 curl -O ftp://selab.janelia.org/pub/software/hmmer3/3.1b1/hmmer-3.1b1.tar.gz
 tar xzf hmmer-3.1b1.tar.gz
 cd hmmer-3.1b1/
-./configure --prefix=/usr && make && make install
+./configure --prefix=${HOME} && make && make install
 
 cd /mnt
```

Comment:
These lines could be the problem; depends on the permissions of the 'mnt' folder. Lets download source code and build inside `${HOME}/src`